Dockerfile

```
@@ -198,6 +198,11 @@ COPY scripts/docker/install_airflow_from_latest_master.sh /scripts/docker/instal
 ARG UPGRADE_TO_NEWER_DEPENDENCIES="false"
 ENV UPGRADE_TO_NEWER_DEPENDENCIES=${UPGRADE_TO_NEWER_DEPENDENCIES}
 
+# By changing CI build epoch we can force reinstalling Airflow from the current master
+# It can also be overwritten manually by setting the AIRFLOW_BUILD_EPOCH environment variable.
+ARG AIRFLOW_BUILD_EPOCH="1"
+ENV AIRFLOW_BUILD_EPOCH=${AIRFLOW_CI_BUILD_EPOCH}
```

Shouldn't it be one of the following two:

```suggestion
ARG AIRFLOW_CI_BUILD_EPOCH="1"
ENV AIRFLOW_CI_BUILD_EPOCH=${AIRFLOW_CI_BUILD_EPOCH}
```

or

```suggestion
ARG AIRFLOW_BUILD_EPOCH="1"
ENV AIRFLOW_BUILD_EPOCH=${AIRFLOW_BUILD_EPOCH}
```

---

Technically it doesn't matter for cache burst but for consistency